Basic-Car-Maintenance/Shared/MainView/Views/MainTabView.swift

```diff
@@ -18,6 +18,7 @@ struct MainTabView: View {
     @AppStorage("lastTabOpen") var selectedTab = TabSelection.dashboard
     @Environment(ActionService.self) var actionService
     @Environment(\.scenePhase) var scenePhase
+    @State private var isShowingAlert = false
```

Comment:
Please change this variable to `isShowingRealTimeAlert` because this is different than just some regular alert / error